airflow/configuration.py

```diff
@@ -221,11 +166,39 @@ def run_command(command):
 # How long before timing out a python file import while filling the DagBag
 dagbag_import_timeout = 30
 
+# The class to use for running task instances in a subprocess
+task_runner = BashTaskRunner
+
+# If set, tasks without a `run_as_user` argument will be run with this user
+# Can be used to de-elevate a sudo user running Airflow when executing tasks
+default_impersonation =
```

Comment:
looks like we can specify users per task and also supply a default. I think this makes it possible to kick the scheduler off as a super user and then have tasks running as a different user? Is that useful to us? cc @gwax